x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/saml/SamlMetadataCommand.java

```diff
@@ -93,6 +93,18 @@ public static void main(String[] args) throws Exception {
         exit(new SamlMetadataCommand().main(args, Terminal.DEFAULT));
     }
 
+    public static EntityDescriptor buildEntityDescriptorFromSamlRealm(SamlRealm samlRealm) throws Exception {
```

Comment:
This should be part of another class, see my comments in TransportSamlSPMetadataAction

Reply:
Done